osu.Game/Localisation/PlayerLoaderStrings.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class PlayerLoaderStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.PlayerLoader";
+
+        /// <summary>
+        /// "This map is in {0} state."
+        /// </summary>
+        public static LocalisableString ThisMapIsInState(LocalisableString arg0) => new TranslatableString(getKey(@"this_map_is_in_state"), @"This beatmap is in {0} state.", arg0);
```

Comment:
If we want to match osu-stable's text...

```suggestion
        /// <summary>
        /// "This map is in a {0} state."
        /// </summary>
        public static LocalisableString ThisMapIsInState(LocalisableString arg0) => new TranslatableString(getKey(@"this_map_is_in_state"), @"This beatmap is in a {0} state.", arg0);
```

Though... Part of me thinks this is still a little bit weird, and it'd be better to be worded as:
- `This map is loved`
- `This map is qualified`

Likely with separate localisations, because I'm not sure how well this translates in the first place.

Reply:
Also, the xmldoc and the effective english text create a discrepancy with the use of `map`/`beatmap` terminology.

Reply:
> Also, the xmldoc and the effective english text create a discrepancy with the use of `map`/`beatmap` terminology.

Care to elaborate? Because I have no idea what "discrepancy" this is talking about.

Reply:
Yes I care, xmldoc says "This map", the string is constructed with "This beatmap", that's the discrepancy.

Reply:
Seems like the localisation analyser doesn't work on strings with args:
![Kapture 2024-03-13 at 22 27 50](https://github.com/ppy/osu/assets/35318437/8935613b-a1f7-4658-bebe-8abf27cb9292)

Reply:
cc @peppy to check that you agree with my take on the wording, since it's been accepted here.

Reply:
I'm still hoping for the direction proposed in https://github.com/ppy/osu/pull/22167#issuecomment-1381281031.

I'm fine with either wording. Probably "is loved" is fine.

Reply:
You mean displaying the warning inline? Then it's either going to be me doing programming design or maybe we can get @arflyte to get something together but not sure how long that'll take as duplicate issues keep trickling in.